test/lib/action_cable/subscription_adapter/solid_cable_test.rb

```diff
@@ -136,6 +136,17 @@ class ActionCable::SubscriptionAdapter::SolidCableTest < ActionCable::TestCase
     end
   end
 
+  test "does not raise error when polling with no Active Record logger" do
+    with_active_record_logger(nil) do
+      subscribe_as_queue("channel") do |queue|
+        assert_nothing_raised do
```

Comment:
Let's move this assertion up a level since the polling happens in between the broadcast and the pop